src/redist/targets/GenerateLayout.targets

```diff
@@ -15,9 +15,9 @@
     <CoreSetupBlobVersion>$(MicrosoftNETCoreAppInternalPackageVersion)</CoreSetupBlobVersion>
 
     <!-- Change these individually to or $(CoreSetupBlobVersion), $(AspNetCoreBlobVersion), or appropriate fixed version depending if corresponding .Ref packages are unpinned. -->
-    <NETCoreAppTargetingPackBlobVersion>$(CoreSetupBlobVersion)</NETCoreAppTargetingPackBlobVersion>
-    <AspNetCoreTargetingPackBlobVersion>$(AspNetCoreBlobVersion)</AspNetCoreTargetingPackBlobVersion>
-    <WindowsDesktopTargetingPackBlobVersion>$(CoreSetupBlobVersion)</WindowsDesktopTargetingPackBlobVersion>
+    <NETCoreAppTargetingPackBlobVersion>3.1.0</NETCoreAppTargetingPackBlobVersion>
+    <AspNetCoreTargetingPackBlobVersion>3.1.0</AspNetCoreTargetingPackBlobVersion>
```

Comment:
I suppose this one was unnecessary. Did we also need to pin ASP.NET or is that correctly not being produced. @dougbu @JunTaoLuo

Reply:
We're not planning to produce the ref pack for 3.1.1

Reply:
My question was more should the ref pack for ASP.NET be pinned

Reply:
I instinctively updated this, then noticed that we did not pin it in version.details correspondingly

Reply:
I suppose for consistency we should pin it

Reply:
I would argue you should actually do the least pinning possible. Then, when I fix up Core-Setup to more completely not produce the NETCoreApp targeting pack, we only need to remove that pin, and we don't have to re-question whether we can also remove the AspNetCore targeting pack pin.

Reply:
OK, the build would nevertheless fail when ASP.NET inserts 3.1.1 (that hasn't happened yet apparently) without this change I just realized, pin or no pin, so we should take this diff.